audiostream/test/ni/media/test_helper.cpp

```diff
@@ -43,34 +43,33 @@ namespace
 bool init()
 {
 #if BOOST_OS_WINDOWS
-    boost::filesystem::path::imbue( std::locale( std::locale(), new std::codecvt_utf8_utf16<wchar_t>() ) );
+    std::filesystem::path::imbue( std::locale( std::locale(), new std::codecvt_utf8_utf16<wchar_t>() ) );
```

Comment:
`std::filesystem::path::imbue()` does not exist.
It's better to not rely on string conversion, but instead always use the native string format.

Reply:
Here I wasn't even sure what this is all about. Could you elaborate why this needed?

Reply:
`boost::filesystem::path::imbue` sets the path locale for `boost::filesystem`. Using the `std::codecvt` facet sets up the proper conversion between UTF8 and UTF16 characters. This is needed on Windows for setting up a proper string conversion, as Windows uses a locale based on the system settings. Leaving the locale up to Windows results in conversions when the paths have special characters possibly failing (consider Japanese and Chinese characters on a western system).

See: https://stackoverflow.com/questions/45572209/getting-a-boostfilesystempath-as-an-utf-8-encoded-stdstring-on-windows

Note: `std::codecvt` has been deprecated so this isn't a futureproof solution.